src/frameworks/uikit/ui_color.rs

```diff
@@ -91,6 +98,15 @@ pub const CLASSES: ClassExports = objc_classes! {
 
 // TODO: more initializers, set methods, more accessors
 
+- (id)initWithWhite:(CGFloat) w alpha:(CGFloat) a {        
```

Comment:
Nitpick: can you remove the space between `(CGFloat)` and `w`, and `(CGFloat)` and `a`, to match other methods?